internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -6159,6 +6159,18 @@ func TestHandleGetWork(t *testing.T) {
 		}(),
 		wantErr: true,
 		errCode: dcrjson.ErrRPCInternal.Code,
+	}, {
+		name:    "handleGetWork: no work during chain reorg",
+		handler: handleGetWork,
+		cmd:     &types.GetWorkCmd{},
+		mockBlockTemplater: func() *testBlockTemplater {
+			templater := defaultMockBlockTemplater()
+			templater.currTemplate = nil
+			templater.currTemplateErr = nil
+			return templater
+		}(),
+		wantErr: true,
+		errCode: dcrjson.ErrRPCInternal.Code,
```

Comment:
It should be `dcrjson.ErrRPCMisc`, but it looks like a different error path is being hit.

Reply:
You're right.  I've updated the test now.  I also added a print in the path in question during testing to confirm it was being hit that way.

Reply:
I also reverted the fix and observed the test panic, so it is indeed hitting the right thing.